.gitignore

```diff
@@ -212,6 +212,11 @@ vendor/
 .srl
 
 # Binaries
-
 authenticator/bin
 authenticator/pkg
+
+# Terraform-related
+terraform/*/.tfvars
+terraform/*/.tfvars.json
+terraform/*/.terraform
+terraform/*/terraform.tfstate*
```

Comment:
I added this to `.gitignore` because folks will probably just want to pull down this Github repo, `cd` into `terraform/aws/`, and then start running `terraform init` and `terraform apply` from there. Being able to do so for testing without accidentally committing extra Terraform management files is convenient.